pkg/registry/networking/rest/storage_settings.go

```diff
@@ -50,11 +50,13 @@ func (p RESTStorageProvider) NewRESTStorage(apiResourceConfigSource serverstorag
 func (p RESTStorageProvider) v1Storage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) (map[string]rest.Storage, error) {
 	storage := map[string]rest.Storage{}
 	// networkpolicies
-	networkPolicyStorage, err := networkpolicystore.NewREST(restOptionsGetter)
+	networkPolicyStorage, networkPolicyStatusStorage, err := networkpolicystore.NewREST(restOptionsGetter)
 	if err != nil {
 		return storage, err
 	}
 	storage["networkpolicies"] = networkPolicyStorage
+	// TODO: Define if we should disable the whole status subresource in case of the FG disabled.
```

Comment:
Not sure if I should feature gate the subresource path here. Maybe feature gating here will help us hide status, etc (as the storage wont be available) but we wont break anything persisted.

Reply:
The goal of a gate is that it behaves as if the feature isn't there.  This case is interesting - if we have saved data and then disable the gate, would we no longer be able to update that data?

Reply:
I think so, but I will test here if hiding is a viable option. Like, if I disable, then re-enable will the last information be there? Will I get some weird error in this case? Let me test a bit